### DISCUSSION TITLE
Invoke-WebRequest : The remote name could not be resolved

### DISCUSSION THREAD

**Discussion by Minus183** [SpotX-Official/SpotX#165] [Errors]
When I run install.bat it shows like this : 
Invoke-WebRequest : The remote name could not be resolved: 'raw.githubusercontent.com'
At line:1 char:90
+ ... Tls12}; & {(Invoke-WebRequest -UseBasicParsing 'https://raw.githubuse ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (System.Net.HttpWebRequest:HttpWebRequest) [Invoke-WebRequest], WebExc
   eption
    + FullyQualifiedErrorId : WebCmdletWebResponseException,Microsoft.PowerShell.Commands.InvokeWebRequestCommand

and I am using spotify v [1.1.94.872]
Any Solution?

**Comment by amd64fox** [comment]
Most of the answers are in the [FAQ](https://telegra.ph/SpotX-FAQ-09-19#I-get-the-error-%22The-remote-name-could-not-be-resolved%22-while-running-the-install)

**Reply by Minus183** [reply]
FAQ site showing can't be reached

**Reply by amd64fox** [reply]
bro use vpn
